Source/Urho3D/IO/FileWatcher.h

```diff
@@ -33,6 +33,24 @@ namespace Urho3D
 
 class FileSystem;
 
+/// File change type.
+enum FileChangeType

```

Comment:
Do we have existing enums with such name style?

Reply:
Good point!  Fixed.